src/System.Private.Xml/tests/System.Private.Xml.Tests.builds

```diff
@@ -3,65 +3,125 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
       <OSGroup>Windows_NT</OSGroup>
       <TestTFMs>netcoreapp1.0;netcore50;net46</TestTFMs>
     </Project>
+    <Project Include="Readers\CharCheckingReader\System.Xml.RW.CharCheckingReader.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+    </Project>
     <Project Include="Readers\CustomReader\System.Xml.RW.CustomReader.Tests.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
I would have expected changes on the  actual test .csprojs that set `NugetTargetMoniker` property to `netstandard1.7` now by default, plus changes in the project.jsons to add netstandard1.7 to the frameworks section. Was that not required by your change?